src/plugins/visualizations/common/expression_functions/vis_dimension.ts

```diff
@@ -54,28 +54,35 @@ export const visDimension = (): ExpressionFunctionDefinition<
     },
     format: {
       types: ['string'],
-      default: 'string',
       help: i18n.translate('visualizations.function.visDimension.format.help', {
         defaultMessage: 'Format',
       }),
     },
     formatParams: {
       types: ['string'],
-      default: '"{}"',
       help: i18n.translate('visualizations.function.visDimension.formatParams.help', {
         defaultMessage: 'Format params',
       }),
     },
   },
   fn: (input, args) => {
     const accessor = findAccessorOrFail(args.accessor, input.columns);
+    const column = typeof accessor === 'number' ? input.columns[accessor] : accessor;
+    const columnFormat = column.meta.params;
+    // if user doesn't specify format and format for column is not specified
+    // that we should use default format id ('string')
```

Comment:
nit: 
`if user doesn't specify format and format for column is not specified that we should use default format id ('string')` -> `if a user hasn't specified the format of the column and its format is not specified at the table columns, then the default format id ('string') should be used`

Reply:
done